configure.ac

```diff
@@ -780,7 +780,7 @@ AS_CASE(["$host_os"],
         ])
 
 AC_SUBST(INIT_EXT)
-AC_DEFINE_UNQUOTED([SUPPORT_PROCFS], [$PROCFS],
+AC_DEFINE_UNQUOTED([HAVE_PROCFS], [$PROCFS],
```

Comment:
FYI this is fine, but the reason I suggested HAVE_LINUX_PROCFS is that some other OSes actually do have a procfs (it originated in Solaris I believe) but they do not follow the same path naming conventions as Linux, which are the only ones Pacemaker supports

Reply:
Not intentional, just on autopilot :)

https://github.com/ClusterLabs/pacemaker/pull/2831